README.md

```diff
@@ -83,11 +83,15 @@ You can change some options like in CLI :
     "_unknown": [
         "--puppeteer-ignoreHTTPSErrors",
         "--puppeteer-slowMo",
-        "20"
+        "20",
+        {"args": ["--no-sandbox", "--disable-setuid-sandbox"]}
```

Comment:
This is a useful thing here. Could you please add a comment to explain what does it do? (I mean, this is the reason you're making this PR)